Updated I18n support and Spanish translation

macagua opened the pull request.

macagua commented:
@nileshgulia1 :-)

macagua commented:
@avoinea please can you check this "pull request" and if all ok merge it? thanks

macagua commented on src/components/manage/Blocks/Accordion/LayoutSchema.js:
```diff
@@ -1,9 +1,119 @@
+
 const Schema = {
-  title: 'Accordion block settings',
+  title: intl.formatMessage(messages.AccordionBlockSettings),
```
@avoinea I am sorry for the delay, please can you check again this "pull request" and if all ok merge it? thanks

macagua replied:
Done with commit [4281bab]

avoinea commented:
@macagua How do you test your changes? Did you start Volto with this branch?!

```
/opt/sandbox/clean-volto-project/node_modules/react/cjs/react.development.js:1476
      throw Error( "Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:\n1. You might have mismatching versions of React and the renderer (such as React DOM)\n2. You might be breaking the Rules of Hooks\n3. You might have more than one copy of React in the same app\nSee https://reactjs.org/link/invalid-hook-call for tips about how to debug and fix this problem." );
            ^

Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://reactjs.org/link/invalid-hook-call for tips about how to debug and fix this problem.

  49 |
  50 | const applyConfig = (config) => {
> 51 |   const intl = useIntl();
  52 |   config.blocks.blocksConfig.accordion = {
  53 |     id: 'accordion',
  54 |     title: intl.formatMessage(messages.Accordion),

    at resolveDispatcher (/opt/sandbox/clean-volto-project/node_modules/react/cjs/react.development.js:1476:13)
    at Object.useContext (/opt/sandbox/clean-volto-project/node_modules/react/cjs/react.development.js:1484:20)
    at useIntl (/opt/sandbox/clean-volto-project/node_modules/react-intl/dist/components/useIntl.js:7:24)
    at applyConfig (/opt/sandbox/clean-volto-project/src/addons/volto-accordion-block/src/index.js:51:1)
    at /opt/sandbox/clean-volto-project/build/webpack:/tmp/tmp-218169-r8Vo36tiOEyv-.js:16:1
    at /opt/sandbox/clean-volto-project/build/webpack:/tmp/tmp-218169-r8Vo36tiOEyv-.js:34:1
    at Array.reduce (<anonymous>)
    at load (/opt/sandbox/clean-volto-project/build/webpack:/tmp/tmp-218169-r8Vo36tiOEyv-.js:34:1)
    at Module../node_modules/@plone/volto/src/config/index.js (/opt/sandbox/clean-volto-project/build/webpack:/node_modules/@plone/volto/src/config/index.js:224:1)
    at __webpack_require__ (/opt/sandbox/clean-volto-project/build/webpack:/webpack/bootstrap:754:1)

sswp> !!! Script exited with code 1
```

nileshgulia1 commented on src/index.js:
```diff
@@ -22,7 +29,7 @@ const extendedSchema = (config) => {
     .filter((val) => !!val);
 
-  choices.push(['accordion', 'Accordion']);
+  choices.push(['accordion', intl.formatMessage(messages.Accordion)]);
```
I suppose how jenkins passes then? 🤔

macagua replied:
Done with commit [4281bab]

ichim-david commented:
@macagua if you can update your fork with the latest changes from devel we can proceed with trying to merge the changes.

macagua commented:
@avoinea @ichim-david I fixed all errors, I tested this addon following the DEVELOP file and updated this branch, I hope this this pull request pass all the test checkout.

macagua commented on src/components/manage/Blocks/Accordion/Schema.js:
```diff
@@ -66,28 +98,28 @@ const messages = defineMessages({
 
 export const AccordionSchema = {
-  title: 'Accordion',
+  title: intl.formatMessage(messages.Accordion),
```
Done with commit [4281bab]

avoinea commented:
@macagua Thank you! 

As I am on vacation this week I will ask someone from EDW to look into your PRs. Otherwise I can take a look next week: 😉

ichim-david commented:
@macagua  have a look at the pull request that I've opened by making a local pull request with the changes from your fork, I will continue with writing there

macagua commented:
@avoinea enjoy your vacation 
@ichim-david ok any news, just tell me...

ichim-david commented:
@macagua  I'm going to close this pull request as I took your commits and made a local branch
where I also had to make some commits in order to get the build to pass.
https://github.com/eea/volto-accordion-block/pull/91
Next week we will also make a release of these changes, thank you for your contribution!

macagua commented:
@ichim-david Thanks your time and advices to help me for make done it. I learned mucho!
